skyVegasTests/stakeUpdateGameClient.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, type Page } from '@playwright/test';
+import {
+    launchGame,
+    login,
+    validateConsoleMessages,
+    startEventListener,
+    readGames,
+} from './utilities';
+import { ExpectedMessage } from '../types/expectedMessage';
+
+let page: Page;
+readGames().forEach((game) => {
+    const consoleMessages: string[] = [];
+
+    test.describe(`Testing with text: ${game}`, async () => {
+        test.beforeAll(async ({ browser }) => {
+            page = await browser.newPage();
+            await login(page);
+        });
+
+        test.afterAll(async () => {
```

Comment:
We should use test.afterEach instead please. This will ensure if more than 1 tests is added in the future to this file , we will perform this action after each one of those tests.